### PR TITLE
Add test to check for duplicate folder creation

### PR DESCRIPTION
**Pull request by tobiasKaminsky**

### CODE REVIEW

File: library/src/androidTest/java/com/owncloud/android/CreateFolderRemoteOperationIT.java [hunk 1]
```diff
@@ -66,6 +68,24 @@ public void testCreateFolder() {
...
+     */
+    @Test
+    public void testCreateDuplicateFolder() {
+        String remotePath = mFullPath2FolderBase + "duplicateFolder";
+        mCreatedFolderPaths.add(remotePath);
+        RemoteOperationResult<String> result = new CreateFolderRemoteOperation(remotePath, true).execute(client);
+        assertTrue(result.isSuccess());
+
+        // Create folder again
+        remotePath = mFullPath2FolderBase + "duplicateFolder";
```
Comment (ZetaTom) [inline library/src/androidTest/java/com/owncloud/android/CreateFolderRemoteOperationIT.java:82]: `remotePath` is already set to this value.

```suggestion
```

**Review by ZetaTom — COMMENTED** [review]
Test works as expected.

**Review by ZetaTom — APPROVED** [review]
